src/index.js

```diff
@@ -273,6 +299,11 @@ Application.prototype.selectImage = function(app) {
             dockerImage = settings.images.open365_office;
     }
 
+    if (dockerImage.split(':').length == 1) {
```

Comment:
Why are you using `split` instead of `indexOf`. You are wasting time and space, and your intentions aren't as clear as they could be.